sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -16,13 +16,14 @@ toc::[]
 - Ensure proper isolation and sandboxing for testing software.
 - Ensure default resource containment through VM boundaries.
 
-Furthermore, {sandboxed-containers-first} provide an additional option for users to choose from the type of workload they want to run to cover a wide variety of use cases.
+{sandboxed-containers-first} also provides users the ability to choose from the type of workload they want to run to cover a wide variety of use cases.
+
+The {sandboxed-containers} enables users to manage the lifecycle of sandboxed containers on a cluster. The operator allows you to perform tasks such as installation and removal, updates, and status monitoring.
```

Comment:
ISG:

> ⚠️ enable _verb_
> Use to describe the action of making an interface control or product function available to users. Avoid stating
that inanimate objects grant abilities to people, as in "the product enables you to...". Whenever possible, use a
direct, user-focused alternative such as "you can use the product to..." or "with this product, users can...".